index.js

```diff
@@ -100,23 +133,31 @@ function myMap(array, callback) {
 // ***** Callbacks - Question 1 *****
```

Comment:
Looks like you missed this one? Here's an example of what creating our own `map` would look like though!

```javascript
function myMap(array, callback) {
  const result = []
  for (let i = 0; i < array.length; i++) {
    const element = array[i]
    const value = callback(element)
    result.push(value)
  }
  return result
}
```